packages/financial-templates-lib/src/price-feed/CoinGeckoPriceFeed.ts

```diff
@@ -53,17 +53,17 @@ class CoinGeckoPriceFeed extends PriceFeedInterface {
     };
   }
 
-  async update() {
-    const currentTime = this.getTime();
```

Comment:
hhmm. was this not using `await` before when calling this?

Reply:
looks like the case in a few places. thanks ts!

Reply:
yep, surprised they were working!!